INSTALL_MACOS.md

```diff
@@ -18,7 +18,7 @@ Everything is compiled using the latest version of [Clang](https://clang.llvm.or
 
 ## Building on a VM (Virtual Machine)
 
-Natron can be built on a virtual machine, in order to target older versions of OS X / macOS.  Due to the OS X Software License Agreement, OS X may only be virtualized on Mac hardware. The excellent blog post [OS X on OS X](https://ntk.me/2012/09/07/os-x-on-os-x/) ([archive](https://web.archive.org/web/20190520141144/https://ntk.me/2012/09/07/os-x-on-os-x/)) gives all the instructions to install an OS X virtual machine.
+Natron can be built on a virtual machine, in order to target older versions of macOS.  Due to the MacOS Software License Agreement, MacOS may only be virtualized on Mac hardware. The excellent blog post [OS X on OS X](https://ntk.me/2012/09/07/os-x-on-os-x/) ([archive](https://web.archive.org/web/20190520141144/https://ntk.me/2012/09/07/os-x-on-os-x/)) gives all the instructions to install a MacOS virtual machine.
```

Comment:
no need to change all that. The original text was ok

Reply:
Fixed and reverted to original.